nnpdf_data/nnpdf_data/commondata/ATLAS_2JET_13TEV_DIF/filter.py

```diff
@@ -91,3 +93,29 @@ def processData():
 
 
 processData()
+
+# add w/o lumi variant of uncerts
+search_block = """
+  syst_lumi:
+    description: ''
+    treatment: MULT
+    type: CORR
+"""
+replace_block = """
+  syst_lumi:
+    description: ''
+    treatment: MULT
+    type: SKIP
+"""
+
+
+def lumiless(source_file, destination_file):
+    shutil.copy(source_file, destination_file)
+    with open(destination_file, 'r') as file:
+        content = file.read()
+    content = content.replace(search_block, replace_block)
+    with open(destination_file, 'w') as file:
+        file.write(content)
+
```

Comment:
Please, thes efunctions which are all equal or very similar, lift them to the utils module.

(Also, instead of running isort and black separately, you can use pre-commit before doing commits!)